src/logic/__tests__/startAccountMigration.test.js

```diff
@@ -0,0 +1,137 @@
+import {Observable} from 'rxjs';
+import reduce from 'lodash-es/reduce';
+
+import reducer from '../../reducers/user';
+import startAccountMigration from '../startAccountMigration';
+import {
+  accountMigrationComplete,
+  accountMigrationError,
+  accountMigrationNeeded,
+  accountMigrationUndoPeriodExpired,
+  startAccountMigration as startAccountMigrationAction,
+  userAuthenticated,
+  dismissAccountMigration,
+} from '../../actions/user';
+import {migrateAccount} from '../../clients/firebase';
+import {getCurrentAccountMigration} from '../../selectors';
+import {bugsnagClient} from '../../util/bugsnag';
+
+import {
+  credentialFactory,
+  firebaseErrorFactory,
+  userFactory,
+} from '@factories/clients/firebase';
+import {githubProfileFactory} from '@factories/clients/github';
+import {firebaseProjectFactory} from '@factories/data/firebase';
+
+jest.mock('../../clients/firebase.js');
```

Comment:
Nit, but I don’t think you need to write the `.js` extension here :slightly_smiling_face: